skiko/src/awtMain/kotlin/org/jetbrains/skiko/redrawer/MetalRedrawer.kt

```diff
@@ -74,7 +79,9 @@ internal class MetalRedrawer(
             setVSyncEnabled(device, enabled = false)
             update(System.nanoTime())
             performDraw()
-            setVSyncEnabled(device, properties.isVsyncEnabled)
+            if (!isDisposed) {
```

Comment:
If it will be easy, could you write a test in SkiaLayerTest where we dispose the layer in draw?